pkg/controller/persistentvolume/framework_test.go

```diff
@@ -471,6 +471,62 @@ func (r *volumeReactor) waitTest() {
 	}
 }
 
+// deleteVolumeWithEvent simulates that a volume has been deleted in etcd and
+// the controller receives 'volume deleted' event.
+func (r *volumeReactor) deleteVolumeWithEvent(volume *api.PersistentVolume) {
```

Comment:
These functions do not really add/delete volumes/claims, they generate fake events. , `addClaim` would be confusing. I'll rename them to `addClaimEvent`.